lib/fungus_toast/games/helpers/random.ex

```diff
@@ -0,0 +1,6 @@
+defmodule FungusToast.Random do
+    @spec random_chance_hit(float()) :: boolean()
+    def random_chance_hit(percent_chance) do
+        Enum.random(0..9999) < percent_chance * 100
```

Comment:
How does this work? Should we do something like `Enum.random(0..100) < percent_chance * 100`?

Reply:
It seems like Enum.random can only do integer math and I wanted to be sure we can have more granular percentages.  E.g. if you have a 5.23% chance of something, then this will turn that into 523/10000 chance (instead of being 5/100).